src/components/home/Categories.jsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import "./style.scss";
+import sportImg from "../../assets/img/dumbbell.jpg";
+import glassesImg from "../../assets/img/glasses.jpg";
+import setUpImg from "../../assets/img/setup.jpg";
+import shoesImg from "../../assets/img/shoes.jpg";
+import sterhoscopeImg from "../../assets/img/sterhoscope.jpg";
+import NutritionImg from "../../assets/img/supplements.jpg";
+import selfCareimg from "../../assets/img/selfCare.jpg";
+
+export default function Categories() {
+  const settings = {
```

Comment:
let's also move this since its kinda a big constant. `SLIDER_SETTINGS` and have it at the top of the file